src/pages/Home.jsx

```diff
@@ -1,22 +1,107 @@
-        </p>
-        <a
-          href="tel:+123456789"
-          className="items-center w-full p-2 my-20 text-lg text-center text-white bg-green-700 rounded-lg font-body md:mt-16 hover:bg-green-600">
-          (123)-456-789
-        </a>
+  const renderWeb = () => {
+    return (
+      <div className='flex flex-row'>
+        <img
+          className='w-1/3'
+          src='/images/hero.png'
+          alt='Freshly mowed vibrant green grass showcasing immaculate grooming. In the foreground, beautiful flower bushes bloom with a variety of white, red, and pink flowers.'
+        />
+        <div className='w-full'>
+          <div className='flex flex-row absolute right-0 top-0 pt-2'>
+            <p className='font-normal font-header text-4xl mr-4'>
+              Call us today!
+            </p>
+            <p className='font-normal font-header text-4xl text-green-700 mr-4'>
+              123.456.7890
+            </p>
+          </div>
+          <div className='flex flex-col mx-auto mt-18 py-36 text-center items-center'>
+            <div className='mx-2'>
+              <BaseHeading level={1} text='Yorkshire Woods Ground Crew' />
+              <p className='font-normal my-6 italic text-4xl text-zinc-500'>
+                Detroit&apos;s Finest Lawn Care
+              </p>
+            </div>
+            <div className='flex flex-row font-normal text-lg md:text-xl lg:text-2xl mb-14 divide-x-2 divide-zinc-500'>
+              <h3 className='px-2'>Personalized Service</h3>
+              <h3 className='px-2'>Jobs Of All Sizes</h3>
+              <h3 className='px-2'>Quality Guaranteed</h3>
+            </div>
+            <div className='flex items-center text-center justify-center bg-green-700 p-2.5 gap-2.5 rounded-lg w-3/5 lg:w-[463px] lg:h-[85px]'>
+              <Link
+                to='#services-section'
+                className='text-2xl lg:text-3xl text-white font-bold'
+              >
+                Our Services
+              </Link>
+            </div>
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  const renderMobile = () => {
+    return (
+      <>
+       <div className='flex bg-white py-2.5'>
+          <p className='font-normal font-header text-xl ml-auto mx-2'>
+            Tap to call!
+          </p>
+          <p className='font-normal font-header text-xl text-green-700 mx-2'>
+            123.456.7890
+          </p>
+        </div>
+      <div className="p-6 bg-white bg-[url('/public/images/hero.png')] bg-no-repeat bg-cover">
+        <div className='flex flex-col items-center justify-center p-6 bg-white bg-opacity-80 rounded-2xl'>
+          <div className='py-5 px-16 gap-2.5'>
+            <BaseHeading level={1} text='Yorkshire Woods Ground Crew' />
+            <hr className='w-[211px] h-px mx-auto mt-4 bg-black border-0 rounded'></hr>
+          </div>
+          <div>
+            <div className='w-[358px] h-[80px] px-8'>
```

Comment:
nitpick: Please don't use the custom pixel sizes. Instead, please discuss with the designer how we can stick to TW's default sizing.

Reply:
Got it. I just messaged the designer about this. Currently waiting for the TW default sizing.